content/docs/dvclive/get-started.md

```diff
@@ -1,134 +1,129 @@
 # Get Started
 
 DVCLive is a Python library for logging machine learning metrics and other
-metadata in simple file formats, which is fully compatible with DVC.
+metadata in simple file formats which are fully compatible with DVC.
 
-<admon type="tip">
+## Set up DVCLive
 
-If you use one of the supported [ML Frameworks](/doc/dvclive/ml-frameworks), you
-can jump directly to its corresponding page.
+First of all, you need to add DVCLive to your Python code in order to start
+logging metrics.
 
-</admon>
-
-## Workflow
-
-### Initialize DVCLive
+In this example we are using [Keras](/doc/dvclive/ml-frameworks/keras) for
+training our model and we can add DVCLive as follows:
 
 ```python
-from dvclive import Live
-
-live = Live()
+# train.py
+from dvclive.keras import DvcLiveCallback
+...
+model.fit(
+  train_dataset, validation_data=validation_dataset,
+  callbacks=[DvcLiveCallback(path="training_metrics")])
 ```
 
-See [`Live()`](/doc/dvclive/api-reference/live) for details.
-
-### Log data
+You can find example snippets and reference of the options available in the
+corresponding page of any other of the supported
+[ML Frameworks](/doc/dvclive/ml-frameworks).
 
-<toggle>
-<tab title="Scalars">
+If you prefer to use the DVCLive API directly, check the
+[API Reference](/doc/dvclive/api-reference).
 
-```python
-live.log("acc", 0.9)
-```
+## Set up DVC
 
-See `Live.log()`.
+In order to visualize the outputs generated by DVCLive, we can set up a DVC
+pipeline.
```

Comment:
Mention this is optional/ recommended?

Reply:
> Mention this is optional/ recommended?

I think we should treat it as mandatory. Given the current capabilities, I don't think it's realistic to expect people using `dvclive` without a `dvc.yaml`.

This might change as we are discussing new "onboarding" workflows, but for now the above is True.

Reply:
- I wouldn't undersell DVCLive features (working on its own) but up to you. Maybe just mention it can also work without DVC?
- "set up a DVC pipeline" -> I'd avoid the pipeline concept here (and in general). We're talking about a single stage that's just a required wrapper for `dvc exp`.

Reply:
I agree with your hunch @daavoo that it's unlikely people want to use DVCLive on its own, but I also agree with @jorgeorpinel that we should briefly mention that DVC is optional. I think it's important for clarity. It currently reads like DVCLive will break if you don't do the DVC setup, and it's unclear why.

Reply:
PTAL to the new version